pkg/kritis/crd/securitypolicy/securitypolicy_test.go

```diff
@@ -109,12 +109,12 @@ func Test_SeverityThresholds(t *testing.T) {
 		{"high", "HIGH", "", []string{"c"}},
 		{"medium", "MEDIUM", "", []string{"h", "c"}},
 		{"low", "LOW", "", []string{"m", "h", "c"}},
-		{"BLOCK_ALL", "BLOCK_ALL", "", []string{"l", "m", "h", "c"}},
-		{"BLOCK_ALL + explicit allow unfixable", "BLOCK_ALL", "ALLOW_ALL", []string{"l", "m", "h", "c"}},
+		{"block all", "BLOCK_ALL", "", []string{"l", "m", "h", "c"}},
+		{"block all fixable, but allow all unfixable", "BLOCK_ALL", "ALLOW_ALL", []string{"l", "m", "h", "c"}},
 		{"explicit allow all", "ALLOW_ALL", "", []string{}},
 		{"allow all but unfixable", "ALLOW_ALL", "BLOCK_ALL", []string{"l_nofix", "m_nofix", "h_nofix", "c_nofix"}},
-		{"medium + high unfixable", "MEDIUM", "HIGH", []string{"h", "c", "c_nofix"}},
-		{"high + medium unfixable", "HIGH", "MEDIUM", []string{"c", "c_nofix", "h_nofix"}},
+		{"medium fixable + high unfixable", "MEDIUM", "HIGH", []string{"h", "c", "c_nofix"}},
+		{"high fixable + medium unfixable", "HIGH", "MEDIUM", []string{"c", "c_nofix", "h_nofix"}},
```

Comment:
There is something confusing over here. if i reading it correct, 
The Maximum Severity allowed is "HIGH",  and maximumSeverityUnfixable is Medium.
If max severity is high, we should not allow package c and c_nofix right?

Reply:
test.want is the list of expected violations, so you are correct. This test case serves to verify a specific behavior for an unusual case, where maxSeverity is higher than maxFixUnavailableSeverity. So:

If maxSeverity=HIGH and maxFixUnavailableSeverity=MEDIUM:
 - disallow "c" and "c_nofix" (as you've mentioned), as CRITICAL is higher than either.
 - disallow "h_nofix", because it is unfixable and higher than maxFixUnavailableSeverity

Reply:
ah ok. got it.